src/data/missions.csv

```diff
@@ -1298,14 +1303,14 @@
 "【GW】GW特別ミッションを30個クリアしよう","[GW] Complete 30 GW special missions"
 "【GW】GW特別ミッションを45個クリアしよう","[GW] Complete 45 GW special missions"
 "【GW】GW特別ミッションを全てクリアしよう","[GW] Complete all GW special missions"
-"【宝塚記念】育成を1回完了しよう",""
-"【宝塚記念】育成を3回完了しよう",""
-"【宝塚記念】育成を5回完了しよう",""
-"【宝塚記念】育成を10回完了しよう",""
-"【宝塚記念】育成で期間中にファン数を50万人獲得しよう",""
-"【宝塚記念】育成で期間中にファン数を150万人獲得しよう",""
-"【宝塚記念】育成で宝塚記念に勝利しよう",""
-"【宝塚記念】限定ミッションをすべてクリアしよう",""
+"【宝塚記念】育成を1回完了しよう","[Takarazuka Kinen] Complete a training once"
+"【宝塚記念】育成を3回完了しよう","[Takarazuka Kinen] Complete a training 3 times"
+"【宝塚記念】育成を5回完了しよう","[Takarazuka Kinen] Complete a training 5 times"
+"【宝塚記念】育成を10回完了しよう","[Takarazuka Kinen] Complete a training 10 times"
+"【宝塚記念】育成で期間中にファン数を50万人獲得しよう","[Takarazuka Kinen] Accumulate 500k fans during the period"
```

Comment:
During "the" period doesn't really give you any info (how long is "the" period? etc), so i'd change this to read "the mission period".